students/gene/project/node_app.py

```diff
@@ -0,0 +1,89 @@
+
+
+from node_db import Node, NodeStorage
+
+
+def main_menu_usage():
```

Comment:
I'd probably just make this a big string, rather than a function. But not that different